assets/shared/styles/wp-components.scss

```diff
@@ -6,7 +6,62 @@
 @import '~@wordpress/base-styles/breakpoints';
 @import '~@wordpress/base-styles/animations';
 @import '~@wordpress/base-styles/z-index';
-@import '~@wordpress/components/src/style';
+
+// It imports the dependencies from @import '~@wordpress/components/src/style',
+// excepts './date-time/style.scss' that contains an issue and we don't use that.
```

Comment:
It fixes an issue while importing a style directly from `node_modules`. in the `./date-time/style.scss` that we don't use.
At some point, it would be interesting to refactor these styles there to not depend on this.